tests/loggers/test_mosaicml_logger.py

```diff
@@ -194,3 +196,88 @@ def test_auto_add_logger(monkeypatch, platform_env_var, access_token_env_var, lo
     # Otherwise, no logger
     else:
         assert logger_count == 0
+
+
+def test_model_initialized_time_logged(monkeypatch):
+    mock_mapi = MockMAPI()
+    monkeypatch.setattr(mcli, 'update_run_metadata', mock_mapi.update_run_metadata)
+    run_name = 'test-run-name'
+    monkeypatch.setenv('RUN_NAME', run_name)
+    trainer = Trainer(model=SimpleModel(),
+                      train_dataloader=DataLoader(RandomClassificationDataset()),
+                      train_subset_num_batches=1,
+                      max_duration='1ep',
+                      loggers=[MosaicMLLogger()])
+    trainer.fit()
+    assert isinstance(mock_mapi.run_metadata[run_name]['mosaicml/model_initialized_time'], float)
+
+
+def test_token_training_progress_logged(monkeypatch, tiny_bert_tokenizer):
+    transformers = pytest.importorskip('transformers')
+    mock_mapi = MockMAPI()
+    monkeypatch.setattr(mcli, 'update_run_metadata', mock_mapi.update_run_metadata)
+    run_name = 'test-run-name'
+    monkeypatch.setenv('RUN_NAME', run_name)
+    pretraining_train_dataset = RandomTextLMDataset(size=8,
+                                                    vocab_size=tiny_bert_tokenizer.vocab_size,
+                                                    sequence_length=8,
+                                                    use_keys=True)
+    collator = transformers.DataCollatorForLanguageModeling(tokenizer=tiny_bert_tokenizer, mlm_probability=0.15)
+    dataloader = DataLoader(pretraining_train_dataset,
+                            batch_size=4,
+                            sampler=dist.get_sampler(pretraining_train_dataset),
+                            collate_fn=collator)
+    trainer = Trainer(model=SimpleTransformerMaskedLM(vocab_size=tiny_bert_tokenizer.vocab_size),
+                      train_dataloader=dataloader,
+                      max_duration='64tok',
+                      loggers=[MosaicMLLogger()])
```

Comment:
why is the Trainer/model instantiation different from the other tests for batch and epoch? Can we make it simple like those ones?